docs/dashboard/how-dashboard-works.mdx

```diff
@@ -0,0 +1,102 @@
+the listen port, enforced TLS, and even disabling the dashboard altogether.
+
+## Sections and menus
+
+As mentioned in the introduction, Netdata automatically organizes all the metrics it collects from your node, and places
+them into **sections** of closely related charts.
+
+The first section on any dashboard is the **System Overview**, followed by **CPUs**, **Memory**, and so on.
+
+These sections populate the **menu**, which is on the right-hand side of the dashboard. Instead of manually scrolling up
+and down to explore the dashboard, it's generally faster to click on the relevant menu item to jump to that position on
+the dashboard.
+
+Many menu items also contain a **submenu**, with links to additional categories. For example, the **Disks** section is often separated into multiple groups based on the number of disk drives/partitions on your node, which are also known as a family.
+
+![Animated GIF of using Netdata's menus and
+submenus](https://user-images.githubusercontent.com/1153921/80832425-7c528600-8ba1-11ea-8140-d0a17a62009b.gif)
+
+## Charts
+
+Every **chart** in the Netdata dashboard is [fully interactive](/docs/dashboard/interact-charts.mdx). Netdata
+synchronizes your interactions to help you understand exactly how a node behaved in any timeframe, whether that's
+seconds or days.
+
+A chart is an individual, interactive, always-updating graphic displaying one or more collected/calculated metrics,
+which are generated by [collectors](/docs/collect/how-collectors-work.md). 
+
+![Animated GIF of the standard Netdata dashboard being manipulated and synchronizing
+charts](https://user-images.githubusercontent.com/1153921/80839230-b034a800-8baf-11ea-9cb2-99c1e10f0f85.gif)
+
+Hover over any chart to temporarily pause it and see the exact metrics values presented as different dimensions. Click
+or tap to stop the chart from automatically updating with new metrics, thereby locking it to a single timeframe.
+Double-click it to resume auto-updating.
+
+Let's cover two of the most important ways to interact with charts: panning through time and zooming.
+
+To pan through time, **click and hold** (or touch and hold) on any chart, then **drag your mouse** (or finger) to the
+left or right. Drag to the right to pan backward through time, or drag to the left to pan forward in time. Think of it
+like pushing the current timeframe off the screen to see what came before or after.
+
+To zoom, press and hold `Shift`, then use your mouse's scroll wheel, or a two-finger pinch if you're using a touchpad.
+
+See [interact with charts](/docs/dashboard/interact-charts.mdx) for all the possible ways to interact with the charts on
+your dashboard.
+
+## Alarms
+
+Many of the preconfigured charts on the Netdata dashboard also come with preconfigured alarms. Netdata has three primary
+alarm states: `CLEAR`, `WARNING`, and `CRITICAL`. If an alarm moves from a `CLEAR` state to either `WARNING` or
```

Comment:
Hello @joelhans , Netdata has more than 3 alarms states, but we send `CLEAR`, `WARNING` and `CRITICAL`, when users access the endpoint `/api/v1/alarms?all` they will also see `UNINITIALIZED`, when Netdata starts to run. I think the ideal would be to say `Netdata sends three alarm states`

Reply:
Those three are primary. All [Alarm statuses](https://learn.netdata.cloud/docs/agent/health/reference#alarm-statuses)./

Reply:
Thanks for the feedback, @thiagoftsm! Updated this section.